src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -78,8 +78,15 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       this.stopPolling();
 
       // Wait 1 min before polling again & do not show error message when encounter rate limit error
-      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
-        error.responseJSON?.errorCode === 'E0000047') {
+      if (
+        !error.responseJSON?.errorIntent &&
+        (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests')
+          || error.responseJSON?.errorCode === 'E0000047')
+      ) {
```

Comment:
Nit:  is this an accepted way to format an `if` in Okta JavaScript?

Reply:
I refactored the whole method so it should be more readable.